unique_toolkit/pyproject.toml

```diff
@@ -19,9 +19,10 @@ pyhumps = "^3.8.0"
 unique-sdk = "^0.9.2"
 numpy = "^1.26.4"
 python-dotenv = "^1.0.1"
-requests = "^2.32.3"
 regex = "^2024.5.15"
 tiktoken = "^0.7.0"
+requests = "^2.32.3"
+httpx = "^0.27.0"
```

Comment:
If httpx is part of the requirements, the other implementations will never really be used right? Is there a way to explicitly select which backend to use?

Reply:
I removed the dependencies to keep it consistent with the usage of the sdk. But there is the possibility to explicit define a client by setting `unique_sdk.default_http_client` but u cannot change the version.

Maybe we add a `unique_sdk.default_async_http_client` to keep sync and async separate.